webapp/tests/karma/ts/services/xml-forms.service.spec.ts

```diff
@@ -1021,6 +1063,12 @@ describe('XmlForms service', () => {
         })
         .catch(err => {
           expect(err.message).to.equal(`The form "${internalId}" doesn't have an xform attachment`);
+          expect(error.callCount).to.equal(1);
+          expect(error.args[0][0]).to.equal('Error in XMLFormService findXFormAttachmentName : ');
+          expect(feedbackService.submit.callCount).to.equal(1);
+          expect(feedbackService.submit.args[0][0]
+            .startsWith('Error in XMLFormService findXFormAttachmentName : '))
+            .to.be.true;
```

Comment:
I think you can use `match` here, and avoid using the "matched" text in the call:
```suggestion
          expect(feedbackService.submit.args[0][0]).to.match(/Error in XMLFormService findXFormAttachmentName/);
```
This applies to the other tests as well.